src/components/menus/ActivityStatusMenu.svelte

```diff
@@ -0,0 +1,93 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import ActivitiesIcon from '@nasa-jpl/stellar/icons/activities.svg?component';
+  import { createEventDispatcher } from 'svelte';
+  import type { ActivityDirectiveValidationStatus } from '../../types/activity';
+  import type { ActivityErrorCounts } from '../../types/errors';
+  import { Status } from '../../utilities/status';
+  import PlanNavButton from '../plan/PlanNavButton.svelte';
+  import ActivityErrorsRollup from '../ui/ActivityErrorsRollup.svelte';
+
+  export let activityErrorCounts: ActivityErrorCounts;
+  export let activityDirectiveValidationStatuses: ActivityDirectiveValidationStatus[] = [];
+  export let compactNavMode: boolean = false;
+
+  const dispatch = createEventDispatcher();
+
+  let invalidActivityCount: number = 0;
+  let totalActivitiesCheckedCount: number = 0;
+  let totalActivityCount: number = 0;
+
+  $: {
+    ({ invalidActivityCount, totalActivitiesCheckedCount, totalActivityCount } =
+      activityDirectiveValidationStatuses.reduce(
+        (prevCounts, validationStatus) => {
+          return {
+            invalidActivityCount:
+              prevCounts.invalidActivityCount + (validationStatus.validations.success === false ? 1 : 0),
+            totalActivitiesCheckedCount:
+              prevCounts.totalActivitiesCheckedCount + (validationStatus.status === 'complete' ? 1 : 0),
+            totalActivityCount: prevCounts.totalActivityCount + 1,
+          };
+        },
+        {
+          invalidActivityCount: 0,
+          totalActivitiesCheckedCount: 0,
+          totalActivityCount: 0,
+        },
+      ));
+  }
+
+  function onClickViewConsole() {
+    dispatch('viewActivityValidations');
+  }
+</script>
+
+<PlanNavButton
+  title={!compactNavMode ? 'Activities' : ''}
+  menuTitle="Activity Status"
+  showStatusInMenu={false}
+  statusBadgeText={`${invalidActivityCount}`}
+  status={activityErrorCounts.all && activityErrorCounts.all > 0 ? Status.Failed : Status.Complete}
+>
+  <ActivitiesIcon />
+  <svelte:fragment slot="metadata">
+    <div class="activity-status-nav-container">
+      <div class="total-count">
+        {totalActivitiesCheckedCount}/{totalActivityCount} activit{totalActivityCount !== 1 ? 'ies' : 'y'} checked
```

Comment:
Is there any way i can force there to be some number of unchecked activities? And how would some activities not be checked yet, would that be caused by the backend not yet running validation?

Reply:
The only way I've been able to see it is just by changing a parameter and quickly checking to see the pending icon in the table. Afaik, there isn't a way unless the validation is somehow forcefully stopped mid check.